docs/architecture/microservices/architect-microservice-container-applications/scalable-available-multi-container-microservice-applications.md

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
could be good to blur out your e-mail on the previous image. Also, not sure if that red lettering with white border has good contrast

Reply:
Done!